lib/MetaCPAN/Server/Model/CPAN.pm

```diff
@@ -11,9 +11,9 @@ extends 'Catalyst::Model';
 with 'CatalystX::Component::Traits';
 
 has esx_model => (
-    is         => 'ro',
-    lazy_build => 1,
-    handles    => ['es'],
+    is      => 'ro',
+    handles => ['es'],
+    default => sub { MetaCPAN::Model->new( es => $_[0]->servers ) },
```

Comment:
In general it would be my preference if everything that required a builder or a default with an anonymous sub were lazy, unless there's a compelling reason for that not to be the case.  Mostly because that gives us more predictability around exactly when things happen.  We could well move steps out of `BUILD` in future and then weird things might happen here.  Or, if something gets added before this in the BUILD that messes with the order of operations we have something else to debug.

So, in general, if the object does the least possible amount of work before anything really needs to happen, that seems like the easier situation to debug.  At the very least, it won't make the logic any _worse_.  :)